src/app/static/static-values.ts

```diff
@@ -225,10 +225,6 @@ export class StaticValues {
       case 'PS':
       case 'iPS':
         return 'FARM';
-      case 'CRV_LINK':
```

Comment:
why? :)

Reply:
This is a mistake, not sure what happened

Reply:
ouch, fixing right now